### PR TITLE
Update ddoc section formatting macros.

### PR DESCRIPTION
**Pull request by quickfur**
Companion PR to https://github.com/dlang/dmd/pull/7834.

### CODE REVIEW

File: spec/ddoc.dd [hunk 1]
```diff
@@ -834,6 +834,10 @@ $(P
     $(TROW $(ARGS $(D DDOC_KEYWORD)), $(ARGS Highlighting of D keywords.))
     $(TROW $(ARGS $(D DDOC_PARAM)), $(ARGS Highlighting of function parameters.))
     $(TROW $(ARGS $(D DDOC_BACKQUOTED)), $(ARGS Inserts inline code.))
+    $(TROW $(ARGS $(D DDOC_AUTO_PSYMBOL)), $(ARGS Automatic highlighting of declaration name to which a particular section is referring in running text.))
```
Comment (wilzbach) [inline spec/ddoc.dd]: I'm not sure here, but wouldn't be "a declaration name" be better?

File: spec/ddoc.dd [hunk 2]
```diff
@@ -834,6 +834,10 @@ $(P
...
+    $(TROW $(ARGS $(D DDOC_AUTO_KEYWORD)), $(ARGS Automatic highlighting of D keywords in running text.))
+    $(TROW $(ARGS $(D DDOC_AUTO_PARAM)), $(ARGS Automatic highlighting of declaration name to which a particular section is referring in running text.))
```
Comment (wilzbach) [inline spec/ddoc.dd]: This only happens in Ddoc comments - not in pure DD files, so maybe clarify as "in documentation text" or "in documentation header of a symbol"?

File: spec/ddoc.dd [hunk 3]
```diff
@@ -834,6 +834,10 @@ $(P
...
+    $(TROW $(ARGS $(D DDOC_AUTO_PSYMBOL_SUPPRESS)), $(ARGS Words for which automatic highlighting has been suppressed by an underscore prefix `_`.))
```
Comment (wilzbach) [inline spec/ddoc.dd]: How about

> Special macro for finding ...

**Comment by ZombineDev** [comment]
@quickfur why did you close this PR?

**Pull request closed without merging**